### ISSUE TITLE
dateFormat break disabledDates feature

### ISSUE THREAD

**Issue by maferland** [Wikiki/bulma-calendar#203]
- [ ] Is this something you can **debug and fix**? Send a pull request! Bug fixes and documentation fixes are welcome.
- [x] Are you running the latest version?
- [x] Are you reporting to the correct repository?

## Bug Report

**Environment**
- Extension version(s): 6.0.7
- Bulma version: 0.7.5
- OS: OSX 0.14.4
- Browser: Probably all, but Safari / Chrome / Firefox
- Are customizing the extension: No

**Current Behavior**
When using `dateFormat: 'DD/MM/YY'`
The `disabledDates` parameter does not work.

**Input Code**
- Initialization code:

```js
    bulmaCalendar.attach('[type="date"]', {
    disabledDates: [
      new Date(new Date().getTime() - 24 * 60 * 60 * 1000),
      new Date(new Date().getTime() + 24 * 60 * 60 * 1000)
    ],
      dateFormat: 'DD/MM/YY',
    });
```

**Additional context/Screenshots**

https://www.loom.com/share/289307b5d7464a949156b5743ab6fd16

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem** via [codepen](https://codepen.io) or similar.

https://codepen.io/maferland/pen/rNBQbwM

**Note**
If the fix is not too deep, just point out the files and I can have a look.

**Comment by Insperio** [comment]
Same problem here. I found it only works correctly with YYYY-MM-DD. All other formats cause it to fail. Even the example on the demo site is broken :)

**Comment by Wikiki** [comment]
Hi,

thanks for reporting this issue. for your information we are working on the next version which should integrates the new date-fns major release and we are working hard to handle date formats more efficiently. we don't have any ETA yet.

**Comment by michael-hack** [comment]
Should be fixed with 6.1.5
